woocommerce/woocommerce-gutenberg-products-block
# Product Gallery: Add animation when large image changes

> [!WARNING]  
> This PR is blocked by woocommerce/woocommerce-blocks#11111 

> [!NOTE]  
> I created a follow-up issue to add E2E tests woocommerce/woocommerce#42192


## What

Fixes woocommerce/woocommerce-blocks#10870. Currently, there isn't any animation when the large image changes.

## Why

This PR implements the animation when the user changes the large image with:
- slider
- thumbnails
- arrows button (in progress - https://github.com/woocommerce/woocommerce-blocks/pull/10938/)

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Edit the Single Product Template.
2. Add the Product Gallery block.
3. Save the template.
4. Visit a product that has multiple images.
5. Change the large image by clicking on the thumbnails and with the slider.
6. Ensure that the animation works correctly.



## Screenshots or screencast






| Before | After |
| ------ | ----- |
|<video src=https://github.com/woocommerce/woocommerce-blocks/assets/4463174/d2b8ab93-9452-4c61-86d6-9851f4af667a />|<video src=https://github.com/woocommerce/woocommerce-blocks/assets/4463174/1016bd13-df97-4e88-b2e2-1766e749277c />|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
